modules/cli/src/integrationTest/java/org/apache/ignite/internal/cli/CliIntegrationTest.java

```diff
@@ -69,6 +69,7 @@ public abstract class CliIntegrationTest extends ClusterPerClassIntegrationTest
             new MetricSource().name("sql.client").enabled(true),
             new MetricSource().name("sql.plan.cache").enabled(true),
             new MetricSource().name("sql.queries").enabled(true),
+            new MetricSource().name("storage.aipersist").enabled(true),
             new MetricSource().name("storage.aipersist.default").enabled(true),
             new MetricSource().name("storage.aipersist.default_aipersist").enabled(true),
             new MetricSource().name("storage.aipersist.checkpoint").enabled(true),
```

Comment:
What do `storage.aipersist.*` mean? Is `default` a storage profile? What will happen if I name a storage profile 'checkpoint', will metrics clash?

Reply:
> What do storage.aipersist.* mean?

Depending on the context, for example, in this PR, a storage metric was added.

> Is default a storage profile?

Yes.

> What will happen if I name a storage profile 'checkpoint', will metrics clash?

More likely.

How do you propose to fix it?

Reply:
I think we should eliminate the possibility for a clash. One way to do it is like this:

1. For metrics under 'engine + storage profile', we add another level after the engine, like 'aipersist.profile.default'
2. We add our own engine metrics right under the engine, like 'aipersist.checkpoint'